src/main/kotlin/com/team4099/robot2023/commands/drivetrain/FollowPathPlannerPathCommand.kt

```diff
@@ -38,8 +41,12 @@ import org.team4099.lib.units.derived.perMeterSeconds
 import org.team4099.lib.units.derived.radians
 import org.team4099.lib.units.perSecond
 
-class FollowPathPlannerPathCommand(val drivetrain: Drivetrain, val path: PathPlannerPath) :
-  Command() {
+class FollowPathPlannerPathCommand(
+  val drivetrain: Drivetrain,
+  val path: PathPlannerPath,
+  useOdoFrame: Boolean = true,
+  var shiftByOdoTField: Boolean = true,
```

Comment:
I feel like these variable names don't make clear what they're actually doing -- I think you want something like `pathFrame: FrameType = FrameType.FIELD`  and `stateEstimateFrame: FrameType = FrameType.ODOM` or something like this. Then if the frame types match, you just select the right state estimate and send it, if not you do frame conversion on each trajectory state